docs/06_how-to-guides/30_key-value-api/kv_map/50_how-to-iterate-kv-map.md

```diff
@@ -41,7 +41,7 @@ class [[eosio::contract]] smartcontract : public eosio::contract {
 };
 ```
 
-## Procedures
+## Procedure
 
 Complete the following steps to implement an action that is iterating through the first N `person` objects in `kv map` and prints their first and last names:
```

Comment:
"that is iterating" to "which iterates/ to iterate " ?

Reply:
matches the overview - doesn't use 'ing'

Reply:
though in this case I don't believe "iterating" is a gerund so "iterating" is ok

Reply:
I changed it anyway